core/art/views.py

```diff
@@ -1289,9 +1342,10 @@ def registerARTTest(request):
                 gitlabid=gitlabid,
                 computingsite=computingsite,
                 status=art_const.TEST_STATUS_INDEX['active'],
+                test_type=test_type,
             )
             insertRow.save()
-            data = {'exit_code': 0, 'message': "Provided pandaid has been successfully registered"}
+            data = {'exit_code': 0, 'PandaID': pandaid, 'message': "Provided pandaid has been successfully registered"}
```

Comment:
I do not see why to used capital letters here, "pandaid" should be OK. And the message may be updated, something like "Test has been registered" or something like that

Reply:
Using pandaid in lower case now.